djstripe/views.py

```diff
@@ -17,7 +17,9 @@
 from .forms import PlanForm, CancelSubscriptionForm
 from .mixins import PaymentsContextMixin, SubscriptionMixin
 from .models import Customer, Event, EventProcessingException, Plan
-from .settings import PRORATION_POLICY_FOR_UPGRADES, subscriber_request_callback
+from .settings import (
```

Comment:
I'm also not sure why this was left like this, but we've since moved to using:

```python
import .settings as djstripe_settings

if djstripe_settings.WEBHOOK_EVENT_CALLBACK:
```
(etc) to make sure there's no confusion between our settings and the django app settings (from which we pull the stripe api keys)